### DISCUSSION TITLE
Server requirements

### DISCUSSION THREAD

**Discussion by joaodlessa** [pimcore/pimcore#12324] [Q&A / Support]
Hello everyone,

I am searching about server requirements to implant PIM. 
I will implant PIM in AWS with 200 users and approximatelly 500.000 products in containers. 
I need to estimate resources for it. 
What is suggesting to calculate or is there some base document for it?
How do I calculate memory, cpu and size that I will need to do it?

**Comment by BlackbitDevs** [comment]
Start with a smaller instance which fulfills [minimum system requirements](https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Requirements.html) and as load increases you can easily change instance type or add resources. Because why would you want to initially use the biggest possible instance type when in the beginning there is low load.
What exactly is necessary for your application depends on what you will want to do with the Pimcore. It is a difference to simply store 500.000 products to look up some data from time to time or if you want to update 100.000 every day, have automation rules which automatically update the products based on certain rules, provide exports to different output channels and have a frontend application with 1M daily visits.
